server/middleware/featureGate.ts

```diff
@@ -37,3 +37,20 @@ export function caseloadGate(prisons: string[], handler: RequestHandler): Reques
     }
   }
 }
+
+/**
+ * Wraps a request handler and returns 404 unless the user’s *active* case load appears in given prisons
+ * If "*" appears in the prisons list, call is always forwarded to request handler
+ */
+export function activeCaseloadGate(prisons: string[], handler: RequestHandler): RequestHandler {
```

Comment:
Oh I see, there are 2 middleware, one checks only the active case load, the other checks any user' case load - sorry didn't see this 👌